### ISSUE TITLE
In Google Chrome, NVDA does not announce the element under the mouse cursor.

### ISSUE THREAD

**Issue by oaron** [nvaccess/nvda#16442]
### Steps to reproduce:
1. I open Google Chrome and navigate to a random website.
2. In the NVDA settings under the Mouse category, I enable the "Mouse tracking" feature.
3. I point the mouse on an element on the website.
### Actual behavior:

NVDA does not announce the element under the mouse. This issue occurs exclusively in this browser; elsewhere (e.g., Mozilla Firefox), it works perfectly.
### Expected behavior:

When mouse tracking is enabled, NVDA announces the element under the mouse cursor.
### NVDA logs, crash dumps and other attachments:
There is no any error message.
#### NVDA installed/portable/running from source:
Installed
#### NVDA version:
Latest stable (NVDA 2024.1)
#### Windows version:
Windows 11 (23H2) 
#### Name and version of other software in use when reproducing the issue:
Google Chrome version 124.0.6367.61
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No, because it had worked for a few days before.
#### If NVDA add-ons are disabled, is your problem still occurring?
Yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes

**Comment by ungjinPark** [comment]
I have this issue too.  I'm using Windows 11 24H2 26058.1400 build version. I checked this issue with NVDA 2023.3.3, 2023.3.4, 2024.1.0 on stable(124) and Canary(126.0.6437.0) versions.

When I put the mouse pointer on web objects, NVDA announces "pane" only.

**Comment by pvagner** [comment]
I have seen similar reports in czech and slovak user community. According to two independent reports NVDA 2024.1 is less accurate when reporting text under the mouse pointer than it used to be with older versions e.g. 2023.3 and earlier.

**Comment by oaron** [comment]
I think I have the solution. In Chrome's settings, we need to reset to the default settings. Once this is done and the browser is restarted, it works properly. Could you let me know if this solution works for you as well?

**Comment by ungjinPark** [comment]
@oaron  Yes, I've reset my browser to default settings right now, and then I tried it. NVDA announces text of an element under the mouse pointer. But, we need to be careful because we don't find out a cause of this issue. So, I will use a Firefox for the time being.

Also, on Microsoft Edge, NVDA speaks a text when the I put mouse pointer on HTML native elements. but, I found the issue that NVDA doesn't speak role=tab elements including single text.

You can try this issue on following link using MS Edge.
https://accessibility-reproductions.vercel.app/tab-layout

**Comment by hwf1324** [comment]
I had this problem two days ago, but strangely enough, I didn't reset Chrome's settings and it's back to normal today as well.
Some of the things I've done are: yesterday I updated Chrome manually, but it didn't restore it. Windows was also updated once today. I'm not sure if Chrome has automatically updated its version in between?

**Comment by Adriani90** [comment]
Yeah it seems to work properly now in Google Chrome after last updates. Closing as works for me.

**Comment by USMCCommO** [comment]
Chrome started working again across multiple computers.
